client/blocks/user-mentions/docs/example-input.jsx

```diff
@@ -3,14 +3,17 @@
 /**
  * External dependencies
  */
-import React from 'react';
+import React, { Component } from 'react';
 
 /**
  * Internal dependencies
  */
 import ExampleInput from './example-input';
 import withUserMentions from '../with-user-mentions';
 
-const UserMentionsExampleInput = ( { onKeyPress } ) => <textarea onKeyPress={ onKeyPress } />;
+// @todo Move ref forwarding to the HOC
+const UserMentionsExampleInput = React.forwardRef( ( props, ref ) => (
```

Comment:
Interesting, this is the first time I've seen someone use `forwardRef`.

Reply:
It's handy! I need access to the DOM node in the child component. I'm hoping to move the `forwardRef` into the higher-order component later: https://github.com/Automattic/wp-calypso/issues/24518.

Reply:
Ah, gotcha; I think I understand this better now.